proposals/lip-0047.md

```diff
@@ -732,3 +732,4 @@ TBA
 [research:validators-module]: https://research.lisk.com/t/introduce-validators-module/317
```

Comment:
again: could we update the links from the research forum to the LIPs repo for all published LIPs? :)

Reply:
true, I thought I did it already